docs/recipes/debugging-with-webstorm.md

```diff
@@ -0,0 +1,19 @@
+# Debugging AVA tests with WebStorm
+
+Starting with version 2016.2, [WebStorm](https://www.jetbrains.com/webstorm/) and other JetBrains IDEs (IntelliJ IDEA Ultimate, PhpStorm, PyCharm Professional, and RubyMine with installed Node.js plugin) allow you to debug AVA tests. 
+
+## Setup
+
+Add a new *Node.js Run/Debug configuration*: select `Edit Configurations...` from the drop-down list on the top right, then click `+` and select *Node.js*.
+
+In the `JavaScript file` field specify the path to AVA in the project's `node_modules` folder: `node_modules/.bin/ava` on OS X and Linux or `node_modules/.bin/ava.cmd` on Windows.
```

Comment:
Thanks so much for providing this recipe! By the way, in my version of Ava the path to the Ava executable was in node_modules/ava/cli.js.

Also (for anyone else coming after me) if your Node root isn't your Webstorm project root you'll need to update the working directory to point to the Node root.